config/settings.yml

```
@@ -16,3 +16,4 @@ gravatar_email: "https://gravatar.com/emails"
 gravatar_url: "https://secure.gravatar.com/avatar/#"
 avt_size: 50
 items_page: 8
+expired_password_time: 2
```

```suggestion
expired_password_time_2_hours: 2
```
em nên đặt tên biến mà ng ta đọc vô là hiểu luôn nhé

---

vâng ạ

---

sửa comment luôn hè

---

ơ vâng ạ, em quên lưu ạ @@